lib/digestFetch.js

```diff
@@ -0,0 +1,223 @@
+/// !-----------------------------------------------------------------------------------------------------------
```

Comment:
`digest-fetch` port requires license and copyright notice (see https://github.com/devfans/digest-fetch/blob/master/LICENSE). Otherwise this looks good to me.